record/record.app.js

```
@@ -174,7 +194,11 @@
             }
```

this else branch should just throw a malformeduri error